### PR TITLE
Add security.txt

### PR DESCRIPTION
**Pull request by LukasReschke**
To test:

```
curl localhost:8081/index.php/.well-known/security.txt -v
```

Ref https://securitytxt.org

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
Good idea!

**Comment by LukasReschke** [comment]
> Good idea!

I know! Must be some Austrian who had this idea 😉

File: apps/settings/lib/WellKnown/SecurityTxtHandler.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+use OCP\Http\WellKnown\IResponse;
+
+class SecurityTxtHandler implements IHandler {
+	public function handle(string $service, IRequestContext $context, ?IResponse $previousResponse): ?IResponse {
+		if ($service !== 'security.txt') {
+			return $previousResponse;
+		}
+
+		$response = "Contact: https://hackerone.com/nextcloud
+Expires: 2021-12-31T23:00:00.000Z
```
Comment (ChristophWurst) [inline apps/settings/lib/WellKnown/SecurityTxtHandler.php:41]: do you plan to extend this manually?

Comment (LukasReschke) [reply apps/settings/lib/WellKnown/SecurityTxtHandler.php:41]: Yes. Dynamic also seems a bit dishonest in case this ever goes wrong 🙈

**Review by nickvergessen — APPROVED** [review]

File: apps/settings/lib/WellKnown/SecurityTxtHandler.php (hunk 1 above)
Comment (nickvergessen) [reply apps/settings/lib/WellKnown/SecurityTxtHandler.php:41]: We can also always extend it when a new stable branch is branched of and up it to "in 1 year"
Then it matches the EOL of the major version?

**Pull request merged**